### DISCUSSION TITLE
Is it possible to turn ArrayData into json?

### DISCUSSION THREAD

**Discussion by bubbajoe** [apache/arrow-rs#6007] [Q&A]
Hello, I want to create a custom json that looks like this:

```
{
 fields: ["c1", "c2", "c3"],
 values: [
   [1, 2, 3, ... ],
   ["1", "2", "3", ... ],
   [1.1, 2.3, 3.5, ... ]
}
```

I was able to do this before, but the utils and APIs have changes quite a bit. Or perhaps, I am looking in the wrong places.

**Comment by Jefffrey** [comment]
I don't believe we have a native way of encoding like that via arrow-json; it'll require some custom logic around our existing interfaces
